.github/workflows/release.yml

```diff
@@ -158,98 +152,33 @@ jobs:
     steps:
       # The GAP to be wrapped is put into gap-$GAP_BUILD_VERSION/
       # The sage-windows script requires the GAP directory to be named this way.
-      #
-      # If the event is a pushed release tag, then we are wrapping a version of
-      # GAP that already has its packages in place and all manuals compiled.
-      # Therefore sage-windows only needs to compile GAP and the packages.
-      #
-      # Otherwise, we wrap a cloned version of GAP.
-      # In none of these cases do we build GAP's manuals, because it's too slow.
-      # If the event is a PR, then we `make bootstrap-pkg-minimal` to save time
-      # (none of the required packages requires compilation).
-      # If the event is a pushed branch, then we `make bootstrap-pkg-full` and
-      # then run the BuildPackages.sh script.  This takes a long time.
 
       - name: "Set some environment variables according to the GitHub context"
         run: |
-          if [[ $GITHUB_REF == refs/tags/v* ]]; then
-            GAPDEV_DIR="gap-dev"
-            COMPILEGAP="make -j2"
-          elif [[ $GITHUB_EVENT_NAME == 'pull_request' ]]; then
-            GAPDEV_DIR="gap-${GAP_BUILD_VERSION}"
-            COMPILEGAP="make -j2 && make bootstrap-pkg-minimal"
-          elif [[ $GITHUB_REF == refs/heads/* ]]; then
-            GAPDEV_DIR="gap-${GAP_BUILD_VERSION}"
-            COMPILEGAP="make -j2 && make bootstrap-pkg-full"
-          else
-            echo "Unrecognised GitHub situation!"
-            exit 1
-          fi
-
-          echo "Using: GAPDEV_DIR=${GAPDEV_DIR} and COMPILEGAP=${COMPILEGAP}"
-          echo "GAPDEV_DIR=${GAPDEV_DIR}" >> $GITHUB_ENV
-          echo "SAGE_RUN_CONFIGURE_CMD=\"cd \$(SAGE_ROOT) && ${COMPILEGAP}\"" >> $GITHUB_ENV
-
-      # In all cases, we currently need to clone GAP for its release scripts.
-      #
-      # If the GitHub event is a pushed release tag, then we compile/wrap a GAP
-      # tarball that we download from the GitHub release that was just created
-      # and uploaded by the preceding 'unix' job.
-      #
-      # Otherwise, we compile and wrap the appropriate GAP development clone
-      # into the installer.
-      #
-      # Alternatively, we could upload the desired GAP tarball via the
-      # 'actions/upload-artifact' action in the preceding 'unix' job, and then
-      # use this in all cases in this 'cygwin' job. This would save some
-      # repeated work and hence time, and unify the code here slightly.
-      #
-      # Furthermore, this would remove the need for the
-      # download_release_archive.py script, and if we would also remove the need
-      # for the upload_file_to_github_release.py script, as described below,
```

Comment:
argh, ok, so we still need to make sure `upload_files_to_github_release.py` is available, or replace it (which I'd prefer anyway).

But also: this mistake of mine was not revealed because this code path is only tested for a release. Bad!

Perhaps we can always perform the upload -- but to where? Not sure this is possible. But perhaps we can do it at least for the scheduled daily jobs: we could let that (re)create a release `nightly` (i.e. delete it if it exists, then tag the commit from which the scheduled workflow is run, and upload there). Then we still notice failures relatively quickly

Reply:
Maybe also should get rid of `make_github_release.py`... I'll give it a spin